mlfinlab/codependence/information_matrix.py

```diff
@@ -0,0 +1,67 @@
+"""
+This script implements the creation of dependence and distance matrix for the
+methods of Information Codependence.
+"""
+
+#Imports
+import numpy as np
+import pandas as pd
+
+from mlfinlab.codependence.information import variation_of_information_score,mutual_info_score
+from mlfinlab.codependence.correlation import angular_distance,absolute_angular_distance,\
+                                              squared_angular_distance,distance_correlation
+
+
+def get_dependence_matrix(df : pd.DataFrame, base_algorithm : str ,n_bins : int = None, normalize : bool = True):
+    """
+    This function returns dependence and distance matrix for the methods of Information Codependence.
```

Comment:
Does it get both the dependence and the distance? I see there is another function titled get distance.